frame/oracle/src/lib.rs

```diff
@@ -981,6 +1003,26 @@ pub mod pallet {
 			Ok(weighted_average)
 		}
 
+		fn depth_of_history(asset_id: T::AssetId) -> usize {
```

Comment:
Curious, why is this a separate function?

Reply:
i removed it